deps/rabbit/src/rabbit_stream_coordinator.erl

```diff
@@ -1256,11 +1256,12 @@ make_ra_conf(Node, Nodes) ->
     Members = [{?MODULE, N} || N <- Nodes],
     TickTimeout = application:get_env(rabbit, stream_tick_interval,
                                       ?TICK_TIMEOUT),
+    MetricsKey = #{<<"raft_cluster">> => atom_to_binary(?MODULE)},
     #{cluster_name => ?MODULE,
       id => {?MODULE, Node},
       uid => UId,
       friendly_name => atom_to_list(?MODULE),
-      metrics_key => ?MODULE,
+      metrics_key => MetricsKey,
```

Comment:
This looks like a breaking change but non-protocol atoms are omitted from labels: https://github.com/rabbitmq/rabbitmq-server/blob/afa28cbdb3af99e60943159a815a669bfa16367a/deps/rabbitmq_prometheus/src/collectors/prometheus_rabbitmq_core_metrics_collector.erl#L500-L504

(Unless something else is reading out of the `ra_metrics` table and expecting this value)